### PR TITLE
build: remove unused yeoman environment dependency from create utils

### PR DESCRIPTION
**Pull request by mohanraj-r**
### What does this PR do?

- remove unused yeoman environment dependency from create utils
-  fix broken link to commands doc

@W-10832373@

### CODE REVIEW

File: packages/templates/src/utils/createUtil.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@
 import * as fs from 'fs';
 import * as path from 'path';
 // @ts-ignore
```
Comment (gbockus-sf) [inline packages/templates/src/utils/createUtil.ts:9]: nit: can you delete this ts-ignore as well?

File: packages/templates/yarn.lock [hunk 2]
```diff
@@ -5151,7 +5151,7 @@ yeoman-environment@^2.9.5:
     untildify "^3.0.3"
     yeoman-generator "^4.8.2"
```
Comment (gbockus-sf) [inline packages/templates/yarn.lock:5152]: this is somewhat confusing to me.  How does it still have `yeoman-generator "^4.8.2"` as a dep here but can be deleted below?

File: packages/templates/yarn.lock [hunk 3]
```diff
@@ -5185,40 +5185,6 @@ yeoman-generator@4.11.0:
...
-    make-dir "^3.0.0"
-    mem-fs-editor "^7.0.1"
-    minimist "^1.2.5"
-    pretty-bytes "^5.2.0"
-    read-chunk "^3.2.0"
-    read-pkg-up "^5.0.0"
-    rimraf "^2.6.3"
-    run-async "^2.0.0"
-    semver "^7.2.1"
-    shelljs "^0.8.4"
```
Comment (gbockus-sf) [inline packages/templates/yarn.lock:5215]: When I pulled down to to verify locally there was a change (to remove shelljs@^0.8.4) to the lock file file on a fresh install.  Can you commit that change?

File: packages/templates/yarn.lock (hunk 2 above)
Comment (gbockus-sf) [reply packages/templates/yarn.lock:5152]: doesn't seem to beef one install 🤷

Comment (mohanraj-r) [reply packages/templates/yarn.lock:5152]: Think it is getting bumped to  4.11.0 following the semantic version range of `^`

**Review by gbockus-sf — CHANGES_REQUESTED** [review]
Just a cleanup item.

File: packages/templates/yarn.lock (hunk 3 above)
Comment (mohanraj-r) [reply packages/templates/yarn.lock:5215]: Done, I see it now too - I wasn't seeing it when I created the PR.

**Review by gbockus-sf — APPROVED** [review]
ty sir

**Pull request merged**